app/ui-react/packages/api/package.json

```diff
@@ -11,29 +11,34 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.1.5",
-    "@testing-library/jest-dom": "^4.0.0",
     "@types/expect": "^24.3.0",
     "@types/file-saver": "^2.0.0",
     "@types/jest": "^24.0.11",
+    "@types/patternfly-react": "*",
     "@types/react": "^16.4.18",
     "@types/react-dom": "^16.0.9",
     "expect": "^24.5.0",
     "jest": "^25.1.0",
     "jest-cli": "^25.1.0",
+    "jest-dom": "^3.1.3",
     "microbundle": "^0.11.0",
+    "monaco-editor": "^0.19.3",
+    "monaco-editor-webpack-plugin": "^1.8.2",
+    "patternfly-react": "^2.29.2",
```

Comment:
This should come out, I'd rather avoid having this dependency as it brings in patternfly 3 which is a step backwards.